pages/ksphere/kommander/1.1.0-beta/troubleshooting/index.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: layout.pug
+title: Troubleshooting
+navigationTitle: Troubleshooting
+menuWeight: 9
+excerpt: Troubleshooting common issues in Kommander UI
+---
+
+This section describes how you can use the `kubectl` CLI to debug unexpected behaviour in Kommander UI.
+
+### I can't access the UI
+
+If you log in to the UI via the `/ops/landing` page and you see a blank page with `Not authorized`, ensure that your user has been assigned appropriate `opsportal` and `kommander` ClusterRoles. See <a href="/ksphere/konvoy/1.5/security/external-idps/rbac/#portal-authorization">Konvoy RBAC Documentation</a> for a list of default roles. Users and groups configured by adding new Identity Providers must have these roles assigned manually.
+
+If you log in to the UI and see a different error screen, ensure the Kommander deployment and pod is healthy by executing the following command:
+
+`kubectl describe deployment -n kommander kommander-kubeaddons-kommander-ui`
+
+### I can't access Kommander UI from Konvoy UI
+
+If there is no link to Kommander in the sidebar of Konvoy UI, ensure that Kommander has been installed on your cluster. You can either check your `cluster.yaml` for an addon named `kommander` or you can check to see if your cluster has the Kommander ClusterAddon:
```

Comment:
We could also mention the Kommander item in the "Apps" dropdown in the app's header bar.